app/api/dao/user.py

```diff
@@ -233,41 +308,42 @@ def update_user_profile(user_id, data):
     @email_verification_required
     def change_password(user_id, data):
         """ Changes the user's password.
-        
+
         Finds the user with the given ID, checks their current password, and then updates to the new one.
-        
+
         Arguments:
             user_id: The ID of the user to be searched.
             data: The user's current and new password.
-        
+
         Returns:
             A message that indicates whether the password change was successful or not and a second element which is the HTTP response code.
-        
+
         """
 
         current_password = data['current_password']
         new_password = data['new_password']
 
         user = UserModel.find_by_id(user_id)
         if user.check_password(current_password):
-            user.set_password(new_password)
-            user.save_to_db()
-            return messages.PASSWORD_SUCCESSFULLY_UPDATED, 201
+            if check_security(new_password):
```

Comment:
can we just use && in 1 if?

Reply:
@annabauza where should we use &&? I didn't get that.